.github/sast_configurations/codeql.yml

```diff
@@ -0,0 +1,9 @@
+name: "CodeQL configurations"
+
+query-filters:
+  - include:
+      tags contain: security
+      precision:
+        - high
+        - very-high
+      security-severity: /([7-9]|10)\.(\d)+/
```

Comment:
It would be good to write up some public justification for these choices (inline comment, commit message, or PR description would be a start). I think you touch on this in one of the PSR reports but we should be able to easily find the reasoning in GitHub too.